.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2
+jobs:
+  build:
+
+    docker:
+      - image: circleci/node:8
+
+    environment:
+      TZ: /usr/share/zoneinfo/America/Los_Angeles
+      YARN_VERSION: 1.2.1
+      TRAVIS_REPO_SLUG: facebook/react
+
+    parallelism: 4
+
+    steps:
+      - checkout
+
+      - run: echo $CIRCLE_COMPARE_URL | cut -d/ -f7
+
+      - run:
+          name: Install Yarn
+          command: |
+            if [[ ! -e ~/.yarn/bin/yarn || $(yarn --version) != "${YARN_VERSION}" ]]; then
+              curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version $YARN_VERSION
```

Comment:
any reason to pin the version? docker/node comes with at least yarn 1.2.1 already

Reply:
I guess `circleci/node:8` uses `yarn@1.3.1`